index.bs

```diff
@@ -2611,9 +2611,9 @@ Short Circuit</h5>
 	and potential members of a Council who are not renouncing their seat
 	confirm it by a vote which results in both of the following:
 	* at least 80% of them vote affirmatively to adopt this resolution
-	* none of them vote against adopting the resolution
+	* none of them voted against adopting the resolution
```

Comment:
This bullet and the one preceding it should either both use `vote` (my preference) or both use `voted`. I'm only suggesting my preference.
```suggestion
	* none of them vote against adopting the resolution
```